R/utils-xportr.R

```diff
@@ -242,14 +243,51 @@ xpt_validate <- function(data) {
     paste("datetime", 7:40, ".", sep = ""),
     paste("yymmdd", 2:10, ".", sep = ""),
     paste("mmddyy", 2:10, ".", sep = ""),
-    paste("ddmmyy", 2:10, ".", sep = "")
+    paste("ddmmyy", 2:10, ".", sep = ""),
+    "E8601DA.",
+    "E8601DA10.",
+    "E8601DN.",
+    "E8601DN10.",
+    "E8601TM.",
+    paste0("E8601TM", 8:15, ".", 0:6),
+    "E8601TZ.",
+    paste("E8601TZ", 9:20, ".", 0:6),
+    "E8601TX.",
+    paste0("E8601TX", 9:20, "."),
+    "E8601DT.",
+    paste0("E8601DT", 16:26, ".", 0:6),
```

Comment:
```suggestion
    paste0("E8601DT", 16:26, "."),
    paste0("E8601DT", 16:26, ".", 0:6),
```

+ update `paste0("E8601DT", 16:26, ".", 0:6),`